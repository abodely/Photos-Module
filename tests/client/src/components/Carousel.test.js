import React from 'react';
import { shallow } from 'enzyme';
import Carousel from '../../../../client/src/components/carousel';


describe('Carousel', () => {
  const wrapper = shallow(<Carousel />);
  test('Should render', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('Should contain div', () => {
    expect(wrapper.find('div')).toExist();
  });
});
