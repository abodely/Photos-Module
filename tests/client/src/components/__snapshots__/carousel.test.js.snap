// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Should render 1`] = `
<div
  className="loading"
>
  Loading...
</div>
`;
